exdb/views.py

```diff
@@ -383,3 +385,33 @@ def get_context_data(self, *args, **kwargs):
         context = super(SearchExperienceResultsView, self).get_context_data(*args, **kwargs)
         context['search_query'] = self.request.GET.get('search', '')
         return context
+
+
+class SearchExperienceReport(CreateView):
+    access_level = 'basic'
+
+    def get(self, *args, **kwargs):
+        if not self.request.GET.get('experiences'):
+            raise Http404
+        pks = json.loads(self.request.GET.get('experiences'))
+        if not pks:
+            raise Http404
+        experiences = Experience.objects.filter(pk__in=pks).prefetch_related(
+            'planners',
+            'recognition',
+            'subtypes',
+            'keywords',
+        )
+        response = HttpResponse(content_type="text/csv")
+        response['Content-Disposition'] = 'attachment; filename="experiences.csv"'
+
+        writer = csv.writer(response)
+        writer.writerow(['Experience Name', 'Status', 'Author', 'Planners', 'Recognition', 'Start Datetime',
+                         'End Datetime', 'Type', 'Subtypes', 'Description', 'Goals', 'Keywords', 'Audience',
+                         'Guest', 'Guest Office', 'Attendance', 'Created At', 'Next Approver', 'Funds', 'Conclusion',
+                         ])
+
+        for experience in experiences:
+            writer.writerow(experience.get_csv_row())
```

Comment:
I recommend that you use [csv.DictWriter](https://docs.python.org/3/library/csv.html#csv.DictWriter) and convert `get_csv_row()` to return a dict instead.